workspace-manager-clienttests/src/main/java/scripts/testscripts/GetWorkspace.java

```diff
@@ -0,0 +1,48 @@
+package scripts.testscripts;
+
+import bio.terra.testrunner.runner.TestScript;
+import bio.terra.testrunner.runner.config.TestUserSpecification;
+import bio.terra.workspace.client.ApiClient;
+import bio.terra.workspace.api.WorkspaceApi;
+import bio.terra.workspace.client.ApiException;
+import bio.terra.workspace.model.WorkspaceDescription;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import scripts.utils.WorkspaceManagerServiceUtils;
+
+import java.util.UUID;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+public class GetWorkspace extends TestScript {
+    private static final Logger logger = LoggerFactory.getLogger(GetWorkspace.class);
+
+    /** Public constructor so that this class can be instantiated via reflection. */
+    public GetWorkspace() {
+        super();
+    }
+
+    @Override
+    public void userJourney(TestUserSpecification testUser) throws Exception {
+        ApiClient apiClient = WorkspaceManagerServiceUtils.getClientForTestUser(testUser, server);
+        WorkspaceApi workspaceApi = new WorkspaceApi(apiClient);
+
+        /**
+         * This GetWorkspace test expects a bogus workspace id resulting in
+         * workspace not found exception and http code 401
+         * **/
+        try {
+            String invalidWorkspaceId = "11111111-1111-1111-1111-111111111111";
+            WorkspaceDescription workspaceDescription = workspaceApi.getWorkspace(UUID.fromString(invalidWorkspaceId));
+            assertThat("GET workspace does not throw not found exception", false);
+        } catch (ApiException apiEx) {
+            logger.debug("Caught exception getting workspace ", apiEx);
+            assertThat("GET workspace throws not found exception", true);
```

Comment:
at minimum, inspect the exception, a la
```
assertThat("GET workspace throws not found exception", apiEx.getCode() == 404);
```
ideally, use some pre-existing assertion/matcher/construct such as JUnit's `assertThrows` instead of manually writing try/catch with multiple assertions.

of course if this test changes to be looking for a 200, this is irrelevant for this test.

Reply:
For now, we need to do `try-catch` manually. We will look at what you have suggested (something similar to a JUnit's assertion framework) as a future enhancement.

Reply:
I don't think we'll need any change to TestRunner for that. You should just be able to import an assertion library, similar to how assertThat is imported in this file.